app/javascript/components/weather-corner.jsx

```diff
@@ -20,19 +17,13 @@ import {
 const CornerWeather = gql`
   fragment CornerWeather on Weather {
     ...CurrentTemp
-    ...CurrentPercip
-    ...HourlyWeather
-    ...MinutelyWeather
-    ...DailyWeather
     ...SunriseSunsetWeather
+    ...CurrentIcon
   }
 
   ${CurrentTemp.fragments.weather}
-  ${CurrentPercip.fragments.weather}
-  ${HourlyTemps.fragments.weather}
-  ${MinutelyWeather.fragments.weather}
-  ${DailyWeather.fragments.weather}
   ${SunriseSunset.fragments.weather}
```

Comment:
Not technically related to this PR, but SunriseSunset data does not seem to be used anywhere in the corner weather.